pwnlib/rop/rop.py

```diff
@@ -1167,32 +1167,51 @@ def find_gadget(self, instructions):
             if tuple(gadget.insns)[:n] == tuple(instructions):
                 return gadget
 
+    def _flatten(self, initial_list):
+        # Flatten out any nested lists.
+        flattened_list = list()
```

Comment:
`list()` is synonymous to `[]` but slower, more verbose and less readable.
```suggestion
        flattened_list = []
```